src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -400,164 +401,173 @@ export class ServerView extends DirMixin(PolymerElement) {
-              </paper-item>
-            </paper-listbox>
-          </paper-menu-button>
-        </div>
-        <div class="server-location" hidden\$="{{!isServerReachable}}">[[serverLocation]]</div>
-      </div>
-      <div class="tabs-container" hidden\$="{{!isServerReachable}}">
-        <div class="tabs-spacer"></div>
-        <paper-tabs selected="{{selectedTab}}" attr-for-selected="name" noink="">
-          <paper-tab name="connections">[[localize('server-connections')]]</paper-tab>
-          <paper-tab name="settings" id="settingsTab">[[localize('server-settings')]]</paper-tab>
-        </paper-tabs>
-      </div>
-
-      <!-- Unreachable server display -->
-      <div class="card-section unreachable-server" hidden\$="{{isServerReachable}}">
-        <img class="server-img" src="images/server-unreachable.png">
-        <h3>[[localize('server-unreachable')]]</h3>
-        <p></p>
-        <div>[[localize('server-unreachable-description')]]</div>
-        <span hidden\$="{{isServerManaged}}">[[localize('server-unreachable-managed-description')]]</span>
-        <span hidden\$="{{!isServerManaged}}">[[localize('server-unreachable-manual-description')]]</span>
-        <div class="button-container">
-          <paper-button on-tap="removeServer" hidden\$="{{isServerManaged}}">[[localize('server-remove')]]</paper-button>
-          <paper-button on-tap="destroyServer" hidden\$="{{!isServerManaged}}">[[localize('server-destroy')]]</paper-button>
-          <paper-button on-tap="retryDisplayingServer" class="try-again-btn">[[localize('retry')]]</paper-button>
-        </div>
-      </div>
-
-      <iron-pages selected="{{selectedTab}}" attr-for-selected="name" hidden\$="{{!isServerReachable}}">
-        <div name="connections">
-          <div class="stats-container">
-            <div class="stats-card transfer-stats card-section">
-              <iron-icon icon="icons:swap-horiz"></iron-icon>
-              <div class="stats">
-                <h3>[[_getFormattedTransferredValue(totalInboundBytes, '0')]]</h3>
-                <p>[[_getFormattedTransferredUnit(totalInboundBytes, 'B')]]</p>
-              </div>
-              <p>[[localize('server-data-transfer')]]</p>
-            </div>
-            <div hidden\$="[[!isServerManaged]]" class="stats-card card-section">
-              <div>
-                <img class="digital-ocean-icon" src="images/do_white_logo.svg">
-              </div>
-              <div class="stats">
-                <h3>[[managedServerUtilzationPercentage]]</h3>
-                <p>/[[_formatBytesTransferred(monthlyOutboundTransferBytes)]]</p>
-              </div>
-              <p>[[localize('server-data-used')]]</p>
+      <iron-pages id="pages" attr-for-selected="id" selected="{{currentPage}}">
+        <outline-server-progress-step id="progressView" localize="[[localize]]"></outline-server-progress-step>
```

Comment:
This HTML is a bit hard to read. The html`` string supports partial templates: https://polymer-library.polymer-project.org/3.0/api/utils/html-tag

Could you move the content for the 3 different pages to their own functions?